lib/daterangepicker.js

```diff
@@ -400,9 +400,9 @@
                 this.container.find('.ranges').show();
             }
 
-            this.oldStartDate = this.startDate.clone();
-            this.oldEndDate = this.endDate.clone();
-            this.oldChosenLabel = this.chosenLabel;
+            this.oldStartDate = this.oldStartDate || this.startDate.clone();
+            this.oldEndDate = this.oldEndDate || this.endDate.clone();
+            this.oldChosenLabel = this.oldChosenLabel || this.chosenLabel;
```

Comment:
since it inits the plugin each time new props are passed in, this feature was broken.